common/app/model/dotcomrendering/DotcomRenderingDataModel.scala

```diff
@@ -545,7 +556,16 @@ object DotcomRenderingDataModel {
 
     val selectedTopics = topicResult.map(topic => Seq(Topic(topic.`type`, topic.name)))
 
+    def getAffiliateLinksDisclaimer(shouldAddAffiliateLinks: Boolean, shouldAddDisclaimer: Boolean) = {
+      if (shouldAddAffiliateLinks && shouldAddDisclaimer) {
```

Comment:
We should probably make a check for `ActiveExperiments.isParticipating(DisableAffiliateLinks)` here too

Reply:
This function takes shouldAddAffiliateLinks as an argument, and shouldAddAffiliateLinks checks for the test participation. So I believe the check is already covered, and we shouldn't need to repeat it?

Reply:
Oh yes, you're right, good catch 👍